modules/fybrik-implicit-copy-batch/values.yaml

```diff
@@ -12,7 +12,7 @@
 # app_cluster:
 labels: {}
 
-image: "ghcr.io/fybrik/mover:latest"
+image: "ghcr.io/fybrik/mover:master"
```

Comment:
Pls, remind me why do you change DOCKER_TAGNAME from "master" to " latest" but here and below from "latest" to "master"

Reply:
@roytman `DOCKER_TAGNAME` should also be 'master' Thanks.  btw, I will need to revisit this pr as this change is part of a new pr that I would like to push before doing release 0.5.0.